src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-group/info-group.component.scss

```diff
@@ -1,21 +1,7 @@
 @import '../../../../defaults';
 
 .info-group-title {
-  margin: 10px 10px 0px 10px;
-  font-size: 1vw;
-}
-
-.underline {
-  border-bottom: 1px solid $color-info-group-underline;
-  margin-top: 0px;
-  margin-left: 10px;
-  margin-right: 10px;
-  margin-bottom: 20px;
-  padding: 0px;
-}
-
-@media (max-width: 767px) {
-  .info-group-title {
-    font-size: 4vw;
-  }
+  margin: 0 0 1.3vw 0.5vw;
```

Comment:
Suggestion:

Reduce this to `margin: 0 0 0.5vw 0.5vw;`

So it will look like:

![screenshot from 2018-10-04 10-38-20](https://user-images.githubusercontent.com/14297426/46465968-f3574b00-c7c1-11e8-90b8-c5e32f5532de.png)

Instead of:

![screenshot from 2018-10-04 10-38-36](https://user-images.githubusercontent.com/14297426/46465978-fd794980-c7c1-11e8-928a-1d837a0ff646.png)

Reply:
@ricardoasmarques Done.